server/config.coffee

```diff
@@ -34,7 +34,7 @@ config =
     ]
 
     plugins: [
-        MODEL_MODULE
+        'cozydb'
```

Comment:
What I mean is that, the MODEL_MODULE var ensured that pouch was used in case of standalone applicaiton. So, is it something removed or cozydb handles it itself?